registry/lombard/calldata-lbtc-redeem-mainnet.json

```diff
@@ -0,0 +1,1494 @@
+{
+  "$schema": "../../specs/erc7730-v1.schema.json",
+  "context": {
+    "$id": "LBTC",
+    "contract": {
+      "deployments": [
+        {
+          "chainId": 1,
+          "address": "0x8236a87084f8B84306f72007F36F2618A5634494"
```

Comment:
This is the proxy contract address (which is the one that a user would interact with). I think that this is fine, considering it looks like `registry/lido/calldata-stETH.json` does the same thing.

Reply:
This is fine as long as it is the same proxy used for all customers.